WLEmptyState/Classes/Extensions/UIColor+WLEmptyState.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIColor+WLEmptyState.swift
+//  WLEmptyState
+//
+//  Created by Jorge Ovalle on 09/11/20.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+
+    static var accent: UIColor = {
+        return UIColor { (UITraitCollection: UITraitCollection) -> UIColor in
+            if UITraitCollection.userInterfaceStyle == .dark {
+                return UIColor.white
+            } else {
+                return UIColor.black
```

Comment:
could it be like below?
```suggestion
                return .white
            } else {
                return .black
```